src/adapters/refract/transformTransactions.coffee

```diff
@@ -20,93 +16,80 @@ module.exports = (transactions, options) ->
   exampleIndex = -1
 
   transactions.forEach((httpTransaction, httpTransactionIndex, httpTransactions) ->
+    httpRequest = httpTransaction.request
+    httpResponse = httpTransaction.response
+
     # Transactions only have 1 request and 1 response
-    httpRequest = _(httpTransaction).httpRequests().first()
-    httpResponse  = _(httpTransaction).httpResponses().first()
+    httpRequestElement = minim.serialiser06.serialise(httpRequest)
```

Comment:
If I understand correctly, the `httpRequestElement` and `httpResponseElement` are ONLY used for comparing if they are the same request?

I wonder if we can do this a bit more efficiently as if there is a large data structure inside the request or response there can be a big performance hit.

Reply:
Perhaps we can filter out the schema and dataStructure content or manually build a smaller representation of the element for comparison purposes.

Reply:
Yeah, this is definitely something I am working on as part of the final PR I was talking about.